app/controllers/application_controller.rb

```diff
@@ -10,11 +10,6 @@
 class ApplicationController < ActionController::Base
   include Vmdb::Logging
 
-  if Vmdb::Application.config.action_controller.allow_forgery_protection
-    db = MiqDatabase.connected? ? MiqDatabase.first : nil
-    protect_from_forgery :secret => db ? db.csrf_secret_token : SecureRandom.hex(64), :except => :csp_report, :with => :exception
-  end
-
```

Comment:
We probably don't, but @martinpovolny might know.  If we don't what a good way to write one?